.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
```

Comment:
This needs a `fail-fast: false` entry, so we can know which specific tests failed.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
Good catch, thanks. 👍